docs/pages/guides/schema-validation.mdx

```diff
@@ -0,0 +1,120 @@
+The operation above creates the schema, but no schema is attached to a room. To attach this schema to a room, you will POST to the rooms endpoint:
+```ts
+POST https://api.liveblocks.io/v2/rooms/{id}/schema
+{
+  "versionId": "some-id"
+}
+```
+## The validation process in practice
+If you would like to work through an example, you can follow along by using the Multiplayer Form example.
+
+The schema we will define for the example is the following:
+```ts
+POST https://api.liveblocks.io/v2/schemas
+
+{
+  "id": "collaborative-form",
+  "name": "Collaborative Form",
+  "data": `
+    type Logo {
+      name: String
+      theme: String
+    }
+    type storage {
+      logo: LiveObject<Logo>
+    }
+  `
+}
+// 200 Response
+{
+   "versionId": "collaborative-form@1",
+   "schemaId": "collaborative-form",
+   "name": "Collaborative Form",
+   "createdAt": "2023-01-22T15:10:25.225Z",
+   "updatedAt": "2023-01-22T15:10:25.225Z"
+}
+```
+
+To attach this schema to a room, you will POST to the rooms endpoint:
+
+```ts
+POST https://api.liveblocks.io/v2/rooms/{id}/schema
+{
+  "schema": "collaborative-form@1"
+}
+```
+
+To simulate the importance of schema validation and also demonstrate how operations can fail, we will change the nested value of `name` to `organization`:
+
+```tsx file="_app.tsx"
+    <RoomProvider
+      id={roomId}
+      initialPresence={{ focusedId: null }}
+      initialStorage={{
+        logo: new LiveObject({
+          organization: "Acme Inc.",
+          theme: "light",
+        }),
+      }}
+    >
+```
+In `index.tsx`, we will update all references from `name` to `organization`: this is a simple but common instance of updating variable names, which could easily break our integration. 
+After updating and running locally, we should now see an error because our deployed schema and client code are out of sync.
+
+<!----Insert sample failure message here, once this is released. You also mentioned demonstrating a refresh, but I have no way to record this currently !--->
+
+When an instance like this occurs, it's indicative of a bug in your app, which you should fix in development, just like how you have to fix a TypeScript bug in your application before shipping it. 
+
+
+<!----I believe the plan here was to insert some components that were used on the comments page --->
+
+We’re opening up a private beta waitlist
+
+CTA to sign up for it, and to let us know their use case for it 
+<!--- -->
+
+## What’s to come
+The ability to add schema validation via API calls is powerful- but we’re not stopping there. In the future, we will expand the ability to add schema validation with the touch of a button. Our road map includes the ability to generate Typescript types from the schema. In addition, schema validation with Liveblocks will consist of online management and editing of your schemas from within the dashboard (and you can currently sign up for the waitlist here too!) 
```

Comment:
One thing that I would add.

Schema are versioned so it's paving the way to build migration tools in the future.

Reply:
Of course.